src/components/MissionsView.js

```diff
@@ -1,18 +1,15 @@
-import React, { useState } from 'react';
+import React from 'react';
+import { useSelector } from 'react-redux';
 import Row from 'react-bootstrap/Row';
-import Container from 'react-bootstrap/Container';
-import { missionData } from '../data/mock';
 import MissionList from './MissionList';
 
 const MissionsView = () => {
-  const [missions] = useState(missionData);
+  const { items } = useSelector((state) => state.missions);
```

Comment:
Please consider `dispatching` the `fechMissionsAsyncThunk` to fetch the `Missions` from the API before reading from the redux store

Reply:
> Please consider `dispatching` the `fechMissionsAsyncThunk` to fetch the `Missions` from the API before reading from the redux store

It's dispatching in the `Layout` component, to avoid dispatching on route change.